package.json

```diff
@@ -11,7 +11,7 @@
     "setup": "npm install && npm run bootstrap",
     "ci:setup": "npm install && npm run ci:bootstrap",
     "bootstrap": "lerna bootstrap",
-    "ci:bootstrap": "lerna bootstrap --since origin/master",
+    "ci:bootstrap": "lerna bootstrap --include-dependencies --include-dependents --since origin/master",
```

Comment:
Adding the `--include-dependencies` and `--include-dependents` flags here were mistakenly left out and didn't match the other CI commands like:

```
"ci:build": "lerna run build --include-dependencies --include-dependents --since origin/master --parallel",
```